apps/desktop-wallet/src/api/apiDataHooks/address/useFetchAddressBalancesAlph.ts

```diff
@@ -28,7 +28,7 @@ interface UseAddressAlphBalancesProps extends SkipProp {
   addressHash: AddressHash
 }
 
-const useAddressAlphBalances = ({ addressHash, skip }: UseAddressAlphBalancesProps) => {
+const useFetchAddressBalancesAlph = ({ addressHash, skip }: UseAddressAlphBalancesProps) => {
```

Comment:
I renamed many hooks so that it looks nicer in the file explorer, the balance hooks are kinda grouped together now:

<img width="350" src="https://github.com/user-attachments/assets/3f31199e-52e1-4e70-a798-cb2bd999925b"/>